### PR TITLE
docs: specify currently supported schema version

### PR DESCRIPTION
**Pull request by francinelucca**

### CODE REVIEW

File: docs/resource-schemas.md [hunk 1]
```diff
@@ -53,6 +53,15 @@ assets:
       content.
 ```
 
+## Stable Schema Version
```
[the remark this answers is not in the document]
Comment (mattrosno) [reply docs/resource-schemas.md:56]: We use sentence case capitalization for pretty much everything in Carbon: https://www.carbondesignsystem.com/guidelines/content/writing-style#capitalization

File: docs/resource-schemas.md [hunk 2]
```diff
@@ -138,7 +147,7 @@ belongs to a library.
 carbon.yml
 
 ```yml
-# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas/carbon-resources.schema.json
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@1.1.5/carbon-resources.schema.json`
```
Comment (mattrosno) [inline docs/resource-schemas.md:150]:
```suggestion
# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@1.1.5/carbon-resources.schema.json
```

File: docs/resource-schemas.md [hunk 3]
```diff
@@ -201,7 +210,7 @@ been deprecated.
...
```
Comment (mattrosno) [inline docs/resource-schemas.md:213]:
```suggestion
# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@1.1.5/carbon-resources.schema.json
```

File: docs/resource-schemas.md [hunk 4]
```diff
@@ -313,7 +322,7 @@ Libraries and assets can specify links to demo sites.
...
```
Comment (mattrosno) [inline docs/resource-schemas.md:325]:
```suggestion
# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@1.1.5/carbon-resources.schema.json
```

**Review by mattrosno — CHANGES_REQUESTED** [review]
Have some extra "`" in the comments, and a suggestion on capitalization to follow the rest of the doc.

**Review by mattrosno — DISMISSED** [review]
Looks great! We can do this in a separate issue, but it would be good to include instructions to install and use the VS Code extension somewhere. Maybe it lives in this new docs section?

https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml

**Comment by francinelucca** [comment]
@mattrosno  added extension rec

**Review by mattrosno — DISMISSED** [review]

File: docs/resource-schemas.md [hunk 5]
```diff
@@ -53,6 +53,20 @@ assets:
...
 ```
 
+## Stable schema version
+
+The current supported stable version of the resource schemas is _v1.1.5_. You can add the yaml
+language server to your YAML files by adding the following line to the top of your files:
+
+```yml
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@1.1.5/carbon-resources.schema.json
+```
```
Comment (jdharvey-ibm) [inline docs/resource-schemas.md:63]: I'd recommend that we only specify support on the major version level and ensure we follow semantic versioning for this package.

File: docs/resource-schemas.md [hunk 6]
```diff
@@ -53,6 +53,20 @@ assets:
...
+
+**_Note:_** If you're using VS Code as your editor you may
+[install](https://code.visualstudio.com/docs/editor/extension-marketplace) a yaml extension, such as
+[YAML](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) for additional
+language support
```
Comment (jdharvey-ibm) [inline docs/resource-schemas.md:68]:
```suggestion
language support.
```

File: docs/resource-schemas.md (hunk 5 above)
Comment (francinelucca) [reply docs/resource-schemas.md]: Done!

**Review by jharvey10 — DISMISSED** [review]

**Review by mattrosno — DISMISSED** [review]

**Review by mattrosno — APPROVED** [review]

**Review by jharvey10 — APPROVED** [review]

**Pull request merged**